### ISSUE TITLE
seed修改后无法保存

### ISSUE THREAD

**Issue by hchufeng** [v2rayA/v2rayA#256]
### Version, approach of installation, operating system

1. What version of v2rayA are you using?

2. What approach of installation do you install V2rayA?

3. What is your operating system?

### Logs：

 ```bash
 # daemon mode
 journalctl -u v2raya | tail -n 50
 
 # docker mode
 docker logs v2raya
 ```
```shell
# Paste running logs here



```

### v2rayA setting and related screenshot
Paste your v2rayA setting and problem related screenshot.

1.V2raya-1.5.3.2

2.Pacman intall the v2raya

3.manjaro , linux-kernel 5.10.61

when the server link is revised ,the seed option is changed to blank, can't save the seed.

**Comment by mzz2017** [comment]
This problem has been fixed.

You can replace your binary with the newer one [from Github Actions Artifact](https://github.com/v2rayA/v2rayA/actions/runs/1270390730), or just waiting for the next release.

Thanks.

**Comment by mzz2017** [comment]
This issue will be closed because it has been fixed in v1.5.4.
